Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceEnrollmentStatusPageWindows10/MSFT_IntuneDeviceEnrollmentStatusPageWindows10.psm1

```diff
@@ -148,6 +148,12 @@ function Get-TargetResource
             Write-Verbose -Message "Could not find an Intune Device Enrollment Configuration for Windows10 with DisplayName {$DisplayName}"
             return $nullResult
         }
+
+        if($getValue -is [Array])
```

Comment:
Based on @NikCharlebois recommendations #3344 , I've added a throw to prevent processing policies with duplicated names